layouts/webinar/single.html

```
@@ -0,0 +1,178 @@
+{{ define "preloads" }}
+  {{ partial "altcha-css.html" . }}
+{{ end }}
+{{ define "main" }}
+  {{- $lang := .Site.Language.Lang }}
+  <section x-data="{feedbackData: {success: false, inProgress: false, errorMessage: ''}, submitData: {webinarId: '{{ .Site.Params.webinar.webinarId }}', firstName: '', lastName: '', email: '', company: '', captcha: null, acceptNewsletter: false}, webinar: {name: '', language: '', dateStart: null, lead: '', learnTitle: '', learnItems: []}, acceptTerms: false, apiForm: null, captchaState: null}" x-init="apiForm = new ApiForm($refs.form, feedbackData, submitData, API_BASE_URL + '/connect/contact/register-webinar'); fetch(API_BASE_URL + '/connect/contact/webinar/' + submitData.webinarId).then(r => r.json()).then(d => { const lang = '{{ $lang }}'; const languageNames = {en: '{{ i18n "webinar_language_en" }}', de: '{{ i18n "webinar_language_de" }}'}; webinar = {name: d.name, language: languageNames[d.language] || d.language, dateStart: d.dateStart.replace(' ', 'T') + 'Z', lead: d.metadata?.lead?.[lang] || '', learnTitle: d.metadata?.learn?.title?.[lang] || '', learnItems: (d.metadata?.learn?.items || []).map(i => i[lang])}; })" class="container py-12">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)